apps/cookbook/src/app/examples/item-example/examples/input/numeric.ts

```diff
@@ -0,0 +1,22 @@
+import { Component } from '@angular/core';
+
+const config = {
+  selector: 'cookbook-item-example-input-numeric',
+  template: `<kirby-item>
+  <kirby-avatar overlay="true" slot="start">
```

Comment:
You can leave out the `overlay` attribute, as kirby-avatar now has a default bg-color:
```suggestion
  <kirby-avatar slot="start">
```